extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RuleApplyRowFilterAndDataMasking.scala

```diff
@@ -81,17 +81,18 @@ class RuleApplyRowFilterAndDataMasking(spark: SparkSession) extends Rule[Logical
       plan: LogicalPlan,
       identifier: TableIdentifier,
       spark: SparkSession): LogicalPlan = {
+    val plugin = SparkRangerAdminPlugin
```

Comment:
unnecessary？

Reply:
Not really. Just to make sure `getFilterExpr` and `getMaskingExpr` are calling to same SparkRangerAdminPlugin instance. And separate them by catalog in future.

Or let me revert it  this time?